packages/block-editor/src/components/media-upload/README.md

```diff
@@ -129,6 +129,15 @@ Only applies if `gallery === true`.
 - Default: `false`
 - Platform: Web
 
+### autoOpenMediaUpload
```

Comment:
Given that this is in the context of the `MediaUpload` component, it might be just as clear to name this `autoOpen`.

Reply:
Makes sense @guarani 😉 I will make the necessary changes.

Reply:
Changes done here 8c9845c